docs/ios.md

```diff
@@ -12,6 +12,11 @@ cargo install cargo-dinghy
 cargo install cargo-dinghy --force
 ```
 
+### Additional Requirements
+
+In order to deploy to iOS one needs to install [ios-deploy](https://github.com/ios-control/ios-deploy). For example with:
+`brew install ios-deploy`
+
```

Comment:
nit: `ios-deploy` is only needed to test on an actual device. It's not needed for iOS simulator use.